transcoder/output/OutputManager.py

```diff
@@ -33,6 +33,11 @@ def output_type_identifier():
         """Returns identifier of output type"""
         raise OutputFunctionNotDefinedError
 
+    @staticmethod
+    def supports_data_writing():
+        """Returns flag indicating if data writes are supported"""
+        return True
```

Comment:
I might default this to `False` or otherwise add this property to `DiagnosticOutputManager` to return `False`, since that OM only supports writes to the console. *or* we can add `-` as an option to `output_path` to treat STDOUT and merge all writes there. However, I think that simply adding this property as false in diag is the way to go for this CL.

Reply:
mostly because the writer designation in the output manager appears to mean first-class schema and data writing to distinct objects in cloud or posix. I think the STDOUT merge is a little much to consider immediately.

Reply:
The property drives whether or not write record is called. So for diag it still does create output for records.